.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Summary
```

Comment:
If this is fine, i'll raise same PR for other providers - with some specific items in checklist for each (for example, beta provider shouldn't have features still in development).